dapp/src/utils/web3.js

```diff
@@ -17,7 +16,7 @@ export function isCorrectNetwork(chainId) {
   if (process.env.NODE_ENV === 'production') {
     return chainId === 1
   } else if (process.env.NODE_ENV === 'development') {
-    return chainId === 1337 || chainId === 31337
+    return chainId === 1 || chainId === 31337
```

Comment:
is that right?

Reply:
Yea, due to limitations in hardhat I can't specify a different chain id for the fork anymore.